.travis.yml

```diff
@@ -2,13 +2,15 @@ language: php
 
 php:
   - 7.2
-  - 7.3
 
 install:
-  - composer install
+  - composer self-update --rollback && composer install
 
 script:
-- ./vendor/bin/phpunit
+- export TCMS_PRODUCT=$TRAVIS_REPO_SLUG
+- export TCMS_PRODUCT_VERSION=$TRAVIS_PULL_REQUEST_BRANCH
+- export TCMS_BUILD=$TRAVIS_BUILD_NUMBER
+- ./vendor/bin/phpunit tests/
```

Comment:
This is missing the configuration for `~/.tcms.conf`, see https://github.com/kiwitcms/tap-plugin/blob/master/.github/workflows/testing.yml#L33

The Travis CI environment defines the following variables: TCMS_API_URL,  TCMS_PASSWORD , TCMS_USERNAME, TCMS_VERIFY_SSL_CERTIFICATES

Reply:
We dont need the tcms.conf, when its missing the plugin will try to read the environment variables. I will remove the TCMS_PRODUCT/BUILD/VERSION as the plugin try to find the TRAVIS variables by itself. At the moment it cant find the PASSWORD variable, I will investigate it.

Reply:
Protected variables are not exposed to PRs from forks. Open this PR from a branch on the main repository.